server/CRUDAPI.md

```diff
@@ -0,0 +1,9 @@
+## CRUD Documentation
+
```

Comment:
This looks good!  Maybe you would want to put in a (super brief) description of each route?  AKA why would you use (GET) api/description/:productId vs (GET) api/description ?  Just a thought! :)